src/presentation/pages/Note.vue

```diff
@@ -1,30 +1,47 @@
 <template>
   <div v-if="note === null">Loading...</div>
-  <Editor
-    v-else
-    ref="editor"
-    :content="note.content"
-    :read-only="!canEdit"
-    @change="noteChanged"
-  />
+  <div v-else>
+    <div class="control__button">
+      <Button
+        class="header__plus"
```

Comment:
unexpected class

Reply:
Maybe just don't specify classes for now?

Reply:
if you don't use it, clear it